setup.py

```diff
@@ -93,7 +93,7 @@ def get_git_sha() -> str:
         "func_timeout",
         "geopy",
         "graphlib-backport",
-        "gunicorn>=20.1.0",
+        "gunicorn>=20.1.0; sys_platform != 'win32'",
```

Comment:
Please make sure you run `pip-compile` after updating this: https://github.com/apache/superset/blob/master/CONTRIBUTING.md#dependencies

Reply:
@craig-rueda I run pip-compile-multi on Mac M1 and nothing

Reply:
Ok, that's probably expected